### PR TITLE
Upload Internal Obligations - default table text

### PR DESCRIPTION
**Pull request by NateHogan**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml [hunk 1]
```diff
@@ -37,6 +37,13 @@
                 </th>
             </tr>
         </thead>
+        <tbody class="govuk-table__body">
+            <tr>
+                <td colspan="2">
+                    <p>There are currently no PCS obligations uploaded</p>
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:43]: there is this no results partial -"~/Areas/Aatf/Views/ManageEvidenceNotes/Overview/_NoResultsPartial.cshtml. Have a look if you can use it. you can simply pass the display message within.

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:43]: As meg says see if can use the no results partial...

Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:43]: That requires a model of type ManageManageEvidenceNoteOverviewViewModel, could create a specific one for ManageObligations?

Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:43]: @JohnPadgettCivica ?

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:43]: Thats fine we can leave it if we can't re-use...

**Pull request merged**